New log file opened at Wed Jul 20 23:48:07 EDT 2016 by 5897@john-VirtualBox
Taking next connection to: 35061
New client attempting to connect on port: 44688-35061
  Address = /127.0.1.1:35061
This client on port 44688 has uuid 95bf30ab-2641-4df4-bc53-e403180c2ab7 configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-95bf30ab-2641-4df4-bc53-e403180c2ab7
Taking next connection to: 35061
New client attempting to connect on port: 45750-35061
  Address = /127.0.1.1:35061
Handshake exception on socket: 45750
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.io.EOFException
	at java.io.DataInputStream.readInt(DataInputStream.java:392)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 35061
New client attempting to connect on port: 45752-35061
  Address = /127.0.1.1:35061
This client on port 45752 has uuid 5aefb170-d1a5-41f3-9210-023e00a5ae7e configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-95bf30ab-2641-4df4-bc53-e403180c2ab7, play-fork-run-5aefb170-d1a5-41f3-9210-023e00a5ae7e
Taking next connection to: 35061
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 35061
